src/v2/ion/ui-schema/ion-string-handler.js

```diff
@@ -88,10 +90,6 @@ const createUiSchemaForString = (ionFormField, remediationForm, transformedResp,
     Object.assign(uiSchema, getCaptchaUiSchema());
   }
 
-  if(ionFormField.name === 'userProfile.countryCode'){
```

Comment:
why to remove this from `createUiSchemaForString` function ?

Reply:
for our new design, we allow admin to select a dropdown list with predefined values or text box display for all country code data type attribute.